policies/api/openapi.yaml

```diff
@@ -6,67 +6,760 @@ servers:
+          description: Agent backend this policy is for. Cannot change once created.
+        policy_data:
+          type: string
+          description: Agent backend specific policy data in yaml format
+          example: "version: \"1.0\"\nvisor:\n    foo: \"bar\""
+        format:
+          type: string
+          example: yaml
+          description: Policy text format needed to specify when a policy is a yaml
+    PolicyPageSchema:
+      type: object
+      properties:
+        policy:
+          type: array
+          minItems: 0
+          uniqueItems: true
+          items:
+            $ref: "#/components/schemas/PolicyObjSchema"
+        total:
+          type: integer
+          description: Total number of items.
+        offset:
+          type: integer
+          description: Number of items to skip during retrieval.
+        limit:
+          type: integer
+          description: Maximum number of items to return in one page.
+      required:
+        - policy
+    ConfigEntrySchema:
+      type: object
+      properties:
+        title:
+          type: string
+          description: Visual title of the config item shown in UI
+        type:
+          type: string
+          description: "Data field type"
+          enum:
+            - string
+            - password
+            - int
+        name:
+          type: string
+          description: The field label used in the JSON config object for this field
+        description:
+          type: string
+          description: A description of the use of this configuration field
+      example:
+        title: Remote host
+        type: string
+        name: remote_host
+        description:
+    PolicyObjSchema:
+      type: object
+      required:
+        - id
+      properties:
+        id:
+          type: string
+          readOnly: true
+          format: uuid
+          description: Unique identifier (UUID)
+        name:
+          type: string
+          description: A unique name label
+          example: my-policy
+        description:
+          type: string
+          description: User description of this Policy
+          example: An example policy
+        tags:
+          type: object
+          description: User defined key/values for organization and searching
+          example:
+            region: eu
+            node_type: dns
+        backend:
+          type: string
+          example: pktvisor
+          description: The policy backend to use. Cannot change once created.
+        ts_created:
+          type: string
+          readOnly: true
+          format: date-time
+          description: Timestamp of creation
+        version:
+          type: integer
+          readOnly: true
+          example: 0
+          description: A monotonically increasing counter starting at 0 on creation and increasing with each policy update.
+        policy:
+          type: string
+          example:
+            value: "version: \"1.0\"\nvisor:\n    foo: \"bar\""
```

Comment:
Only policy_data has a yaml format, policy it's a json, you should change do json format